public/index.css

```diff
@@ -22,6 +22,38 @@ body {
   padding: 0 3%;
 }
 
+.faq-container {
+  display: flex;
+  flex-direction: column;
+  padding: 0 25%;
+}
+
+.faq-item {
+  padding: 0 3rem;
+}
+
+/* accordion styles */
+.accordion {
+  cursor: pointer;
+  width: 100%;
+  text-align: left;
+  margin: 0.5rem 0;
+  transition: 0.4s;
+}
+
+/* .active,
+.accordion:hover {
+  background-color: #ccc;
+} */
+
+.panel {
+  padding: 0 0 0 1.5rem;
+  background-color: white;
```

Comment:
The background color is `white` by default. Let's remove this line.

Reply:
line removed